crates/storage/src/bound.rs

```diff
@@ -17,16 +21,20 @@ pub enum RawBound {
 pub enum Bound<K> {
     Inclusive(K),
     Exclusive(K),
-    InclusiveRaw(Vec<u8>),
-    ExclusiveRaw(Vec<u8>),
 }
 
 impl<K> Bound<K> {
-    pub fn inclusive<T: Into<K>>(t: T) -> Self {
+    pub fn inclusive<T>(t: T) -> Self
+    where
+        T: Into<K>,
```

Comment:
From now, I want to use the `where` syntax everywhere in the codebase, for consistency.